### ISSUE TITLE
Content Text Bubbles Tour

### ISSUE THREAD

**Issue by yeehaa123** [Unacademic/unacademic-frontend#22]
@ReChavers The tour now works as discussed. The bubbles now need a title, and the text should be formatted in markdown. Can you do that?

**Comment by ReChavers** [comment]
I think something broke with the tour after the first Waypoints level?

**Comment by yeehaa123** [comment]
No, I did. Fixed it. Already... But could you add more specific titles to the tour. Afterwards we can send out the invites.
